mat64/inner.go

```diff
@@ -27,6 +27,17 @@ func Inner(x []float64, A Matrix, y []float64) float64 {
 	var sum float64
 
 	switch b := A.(type) {
+	case RawSymmmetricer:
```

Comment:
Thanks for the catch

This particular case is weird. The main matrix-type interfaces are Matrix and Symmetric. All of the other interfaces follow the normal _er convention, including RawMatrixer. RawSymmetricer follows that precedent.